.github/workflows/prepare-release.yml

```
@@ -0,0 +1,154 @@
+name: Automation - Prepare Release
+
+on:
+  workflow_dispatch:
+    # Determine the version number that will be assigned to the release. During the beta phase, we increment
+    # the minor version number and set the patch number to 0.
+    inputs:
+      candidate-stable:
+        required: true
+        description: Release candidate version (stable)
```

Can you add an example, to make abundantly clear which kind of version this is about?

```suggestion
        description: Release candidate version (stable, like v1.0.0-rc4)
```